pom.xml

```diff
@@ -150,15 +150,16 @@
 					<version>3.2.0</version>
 				</plugin>
 
-				<plugin>
+					<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-checkstyle-plugin</artifactId>
-					<version>${checkstyle.plugin.version}</version>
```

Comment:
`${checkstyle.plugin.version}` is still defined as `2.17`. Why not still reference the property?